mne/gui/_file_traits.py

```diff
@@ -288,7 +303,12 @@ def _get_n_omitted(self):
 
     @cached_property
     def _get__info(self):
-        if self.file:
+        if not self.file:
+            return
+        elif self.file.endswith('.mff'):
```

Comment:
I would make the pattern `self.file.endswith('.fif') or self.file.endswith('.fif.gz'):` and in that case use `return read_info(...)`, else use `return read_raw(...).info`. This should make it work with any reader that supports digitization